### PR TITLE
Updating labels for candidate status filter

### PR DESCRIPTION
**Pull request by noahmanger**
This makes the language changes discussed here https://github.com/18F/openFEC-web-app/issues/859

image

@onezerojeremy can this get on the list for testing friday?

### CODE REVIEW

**Comment by noahmanger** [comment]
To clarify the point on testing: this is kind of a weird language issue and we just want to make sure that it makes sense to folks.

File: openfecwebapp/constants.py [hunk 1]
```diff
@@ -209,11 +209,10 @@
 ])
 
 candidate_status = OrderedDict([
-    ('C', 'Statutory candidate'),
+    ('C', 'For selected election cycle'),
```
Comment (jmcarp) [inline openfecwebapp/constants.py]: Maybe too nit-picky, but I'm not sure this is right. First of all, there isn't always a selected election cycle. Second, the candidate records are using the *most recent* candidate status value, not the candidate status value(s) of the selected cycle(s), if any.

Comment (noahmanger) [inline openfecwebapp/constants.py]: Oh thanks for catching this. @emileighoutlaw and @amykort what do you think?

**Comment by onezerojeremy** [comment]
@noahmanger yes of course!

File: openfecwebapp/constants.py (hunk 1 above)
Comment (jwchumley) [inline openfecwebapp/constants.py]: Not sure I understand the context.

Comment (noahmanger) [inline openfecwebapp/constants.py]: Sorry, original discussion in this issue: https://github.com/18F/openFEC-web-app/issues/859

**Comment by onezerojeremy** [comment]
From initial usability testing: 
No one had any problem filter by candidates who had raised more than 5k, though in retrospect it's clear that the test was not testing the thing it really needed to test, which was "MORE" dropdown, which included options that are not as clear. We aren't sure why these options are needed.   


see also #859

**Comment by jmcarp** [comment]
Ping @noahmanger et al.: where's this at?

**Comment by emileighoutlaw** [comment]
Thanks for pinging @jmcarp! We still have a lingering question:

If the candidate records (past, future, present) are always the most recent status value, why do we need these three options? Why not just have "candidates who raised more than $5,000" regardless of the cycle? Who would use the past/present/future filters, and how?

**Comment by noahmanger** [comment]
Ping ping ping

**Comment by emileighoutlaw** [comment]
:/ I still need an answer to:

If the candidate records (past, future, present) are always the most recent status value, why do we need these three options? Why not just have "candidates who raised more than $5,000" regardless of the cycle? Who would use the past/present/future filters, and how?

**Comment by onezerojeremy** [comment]
I agree with @emileighoutlaw

**Comment by noahmanger** [comment]
Still waiting on the API addition for this.

**Comment by AmyKort** [comment]
I am out of the office until 02/18/2016.




Note: This is an automated response to your message  "Re: [openFEC-web-app]
Updating labels for candidate status filter (#961)" sent on 2/12/2016
5:27:28 PM.

This is the only notification you will receive while this person is away.

**Comment by noahmanger** [comment]
This PR is so out of date I'm going to close it. Once the API update is done we can start fresh.

**Pull request closed without merging**